content/api/vision.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Vision
+---
+
+The main way your robot will sense the outside world is through the webcam which is used to read the AprilTag markers.
+
+{{% notice info %}}
```

Comment:
This is already defined at the intro to the API section, it's not needed here

Reply:
Fair enough, but I feel the page needs an introductory sentence :) I'll remove the part about the AprilTags..